packages/api/src/generated/schema.graphql

```diff
@@ -1248,16 +1249,40 @@ enum RecommendErrorCode {
   UNAUTHORIZED
 }
 
+type RecommendHighlightsError {
+  errorCodes: [RecommendHighlightsErrorCode!]!
+}
+
+enum RecommendHighlightsErrorCode {
+  BAD_REQUEST
+  NOT_FOUND
+  UNAUTHORIZED
+}
+
+input RecommendHighlightsInput {
+  groupIds: [ID!]!
+  highlightIds: [ID!]!
+  note: String
+  pageId: ID!
+}
+
+union RecommendHighlightsResult = RecommendHighlightsError | RecommendHighlightsSuccess
+
+type RecommendHighlightsSuccess {
+  success: Boolean!
+}
+
 input RecommendInput {
   groupIds: [ID!]!
   note: String
   pageId: ID!
+  recommendedWithHighlights: Boolean
```

Comment:
@jacksonh There is `recommendedWithHighlights` in the `RecommendInput`

Reply:
Ah i see, so from the client perspective I would probably always use that option. Will probably add a toggle on the share box that people can use to enable it.

Reply:
So you want me to remove the recommendHighlight API?

Reply:
@sywhb I don't think that we need it now, I can see how maybe in the future we might want to share a single highlight or something though.

Its probably ok to leave it in.

Reply:
Cool. We can merge it now